drivers/ethernet/dsa_ksz8794.c

```diff
@@ -695,17 +695,13 @@ static void dsa_delayed_work(struct k_work *item)
 
```

Comment:
Please change the commit subject of "fix: dsa: ...." to "net: dsa: Initialize ...."